hazelcast/src/main/java/com/hazelcast/spi/NodeEngine.java

```diff
@@ -57,7 +57,7 @@
      *
      * @return the ClusterService.
      */
-    ClusterService getClusterService();
+    InternalClusterService getClusterService();
```

Comment:
Indeed. Another example of broken API where internal data-structures like into the outside like InternalPartitionService.